src/compile/selection/interval.ts

```diff
@@ -84,21 +64,10 @@ const interval:SelectionCompiler = {
     }
 
     const update = {
-      x: extend({}, xi !== null ?
-        {scale: model.scaleName(X), signal: `${name}[${xi}].extent[0]`} :
-        {value: 0}),
-
-      x2: extend({}, xi !== null ?
-        {scale: model.scaleName(X), signal: `${name}[${xi}].extent[1]`} :
-        {field: {group: 'width'}}),
-
-      y: extend({}, yi !== null ?
-        {scale: model.scaleName(Y), signal: `${name}[${yi}].extent[0]`} :
-        {value: 0}),
-
-      y2: extend({}, yi !== null ?
-        {scale: model.scaleName(Y), signal: `${name}[${yi}].extent[1]`} :
-        {field: {group: 'height'}})
+      x: extend({}, xi !== null ? {signal: `${name}_x[0]`} : {value: 0}),
```

Comment:
Why do we need `extend` for these lines? 

It seems like `xi !== null ? {signal: `${name}_x[0]`} : {value: 0}` would be sufficient for the first line and same for others?